improver_tests/metadata/test_check_datatypes.py

```diff
@@ -33,139 +33,44 @@
 import unittest
 
 import numpy as np
-from iris.coords import AuxCoord
 from iris.tests import IrisTest
+from iris.cube import CubeList
 
 from improver.metadata.check_datatypes import (
-    _check_units_and_dtype, _construct_object_list, check_cube_not_float64,
-    check_datatypes, check_time_coordinate_metadata)
+    check_mandatory_standards, check_units)
 
 from ..set_up_test_cubes import (
     set_up_percentile_cube, set_up_probability_cube, set_up_variable_cube)
 
 
-class Test_check_cube_not_float64(IrisTest):
+class Test_check_mandatory_standards(IrisTest):
```

Comment:
You could add tests to show this ignores the things it is meant to (like a string coordinate) and a test to show it checks units too.